packages/tracing/src/utils.ts

```diff
@@ -41,9 +72,56 @@ export function extractTraceparentData(traceparent: string): TraceparentData | u
       parentSpanId: matches[2],
     };
   }
+
   return undefined;
 }
 
+type TracestateHeaderData = { sentry?: string; thirdparty?: string };
+
+/**
+ * Extract data from an incoming `tracestate` header
+ *
+ * @param header
+ * @returns Object containing data from the header
+ */
+export function extractTracestateData(header: string): TracestateHeaderData {
+  let sentryEntry, thirdPartyEntry;
+
+  if (header) {
+    let before, after;
+
+    // find sentry's entry, if any
+    const sentryMatch = SENTRY_TRACESTATE_ENTRY_REGEX.exec(header);
+
+    if (sentryMatch !== null) {
+      sentryEntry = sentryMatch[1];
+
+      // remove the commas after the split so we don't end up with `xxx=yyy,,zzz=qqq` (double commas) when we put them
+      // back together
+      [before, after] = header.split(sentryEntry).map(s => s.replace(/^,*|,*$/g, ''));
```

Comment:
```suggestion
      [before, after] = header.split(sentryEntry).map(s => s.replace(/,/g, ''));
```

This should be good enough I guess. We don't allow for commas anywhere in the middle of a value anyway.

Reply:
Actually, no. That's what I had originally, but if the header comes in with a value like

`x=y,sentry=blah,a=b,c=d`

then getting rid of all commas gets rid of the comma between `a=b` and `c=d`, which isn't what we want.